code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -449,42 +449,55 @@
 
 /obj/item/device/dogborg/sleeper/compactor/afterattack(var/obj/item/target, mob/living/silicon/user, proximity)//GARBO NOMS
 	hound = loc
-	if(!proximity)
-		return
-	if(target.anchored)
-		return
-	if(length(contents) > 32)
-		user << "<span class='warning'>Your [src.name] is full. Eject or process contents to continue.</span>"
+	if(isobj(target))
+		if(length(contents) > 11)//grow that tum after a certain junk amount
+			hound.sleeper_r = 1
```

Comment:
Probably should not update fields an icons before the proximity/anchored/etc checks are performed.
Also, setting sleeper_r = 1 will be overwritten by the next call to update_patient() because it doesn't take contents length into account.